preload.js

```diff
@@ -440,14 +441,7 @@ if (
   };
   /* eslint-enable global-require, import/no-extraneous-dependencies */
   window.lokiFeatureFlags = {};
-  window.lokiSnodeAPI = {
-    refreshSwarmNodesForPubKey: () => [],
-    getFreshSwarmNodes: () => [],
-    updateSwarmNodes: () => {},
-    updateLastHash: () => {},
-    getSwarmNodesForPubKey: () => [],
-    buildNewOnionPaths: () => [],
-  };
+  window.lokiSnodeAPI = {}; // no need stub out each function here
```

Comment:
Do integration tests still work this this change?

Reply:
yes
```
RMacPro:lm-n-fileproxy user2$ yarn test-integration
yarn run v1.17.3
$ ELECTRON_DISABLE_SANDBOX=1 mocha --exit --timeout 10000 integration_test/integration_test.js


  Window Test and Login
    ✓ registration: opens one window
    ✓ registration: window title is correct
    ✓ registration: can restore from seed (11046ms)
    ✓ registration: can create new account (7735ms)
    ✓ registration: can delete account when logged in (16418ms)

  Open groups
    ✓ openGroup: works with valid open group url
    ✓ openGroup: cannot join two times the same open group
    ✓ openGroup: can send message to open group (15590ms)

  Add friends
    ✓ addFriends: can add a friend by sessionID (8189ms)

  Link Device
    ✓ linkDevice: link two desktop devices
    ✓ linkDevice: unlink two devices (14640ms)

  Closed groups
    ✓ closedGroup: can create a closed group with a friend and send/receive a message (9899ms)
```